### ISSUE TITLE
[1.16.4] Right-clicking on Research Module - Experience crashes the game

### ISSUE THREAD

**Issue by qrss** [mcenderdragon/Futurepack-API#727]
Forge version: 35.1.28
Futurepack version: 21.0.3

When you right-click on the T3 research module, the game crashes (java.lang.NullPointerException: Rendering screen).

[crash-2021-01-01_01.13.03-client.txt](https://github.com/mcenderdragon/Futurepack-API/files/5758500/crash-2021-01-01_01.13.03-client.txt)
[debug.log](https://github.com/mcenderdragon/Futurepack-API/files/5758501/debug.log)
[FP-Latest.log](https://github.com/mcenderdragon/Futurepack-API/files/5758502/FP-Latest.log)
[latest.log](https://github.com/mcenderdragon/Futurepack-API/files/5758503/latest.log)

**Comment by mcenderdragon** [comment]
```
java.lang.NullPointerException: Rendering screen
	at net.minecraft.client.gui.AbstractGui.func_238469_a_(SourceFile:198) ~[?:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B}
	at futurepack.common.gui.PartRenderer.drawQuadWithTexture(PartRenderer.java:60) ~[futurepack:1.16.4-21.0.3] {re:classloading}
	at futurepack.common.gui.PartRenderer.renderExp(PartRenderer.java:337) ~[futurepack:1.16.4-21.0.3] {re:classloading}
	at futurepack.common.gui.inventory.GuiModulT3.func_230450_a_(GuiModulT3.java:52) ~[futurepack:1.16.4-21.0.3] {re:classloading}
	at net.minecraft.client.gui.screen.inventory.ContainerScreen.func_230430_a_(ContainerScreen.java:88) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at futurepack.common.gui.inventory.GuiModulT3.func_230430_a_(GuiModulT3.java:41) ~[futurepack:1.16.4-21.0.3] {re:classloading}
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:290) ~[forge:?] {re:classloading}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:491) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:976) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:shutupexperimentalsettings.mixins.json:client.MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:shutupexperimentalsettings.mixins.json:client.MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) ~[1.16.4-forge-35.1.28.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_202] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_202] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_202] {}
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_202] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.16.4-35.1.28.jar:35.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {}
```

**Comment by mcenderdragon** [comment]
Should be fixed in [21.0.4 ](https://www.mediafire.com/file/mmmrw65ckdc34xn/Futurepack-1.16.4-21.0.4.jar/file)

**Comment by qrss** [comment]
Yes, seems to be fixed. Thanks!
